.github/workflows/release.yml

```diff
@@ -0,0 +1,64 @@
+---
+name: release
+
+permissions:
+  contents: read
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch_specifier:
+        description: The branch to release ex. main or 0.6.
+        required: true
+        default: "main"
+        type: string
+
+      target_specifier:
+        description: Specify which repo to deploy, all by default.
+        required: true
+        default: "all"
+        type: choice
+        options:
+          - "all"
+          - "mavenCentral"
+          - "pluginPortal"
+
+      version_override_specifier:
+        description: If set, this version will override the agent's gradle.properties version
+        required: false
+        type: string
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - id: buildkite
+        name: Run Release
+        uses: elastic/apm-pipeline-library/.github/actions/buildkite@current
+        with:
+          vaultUrl: ${{ secrets.VAULT_ADDR }}
+          vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
+          vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
+          pipeline: observability-release
+          waitFor: false
+          printBuildLogs: false
+          buildEnvVars: |
+            repo=${{ github.repository }}
+            commit=${{ github.sha }}
+            command=.ci/release.sh
+            channel='#apm-agent-mobile'
+            branch_specifier=${{ inputs.branch_specifier || 'main' }}
+            target_specifier=${{ inputs.target_specifier || 'all' }}
+            version_override_specifier=${{ inputs.version_override_specifier || '' }}
+
+      - if: ${{ always() }}
```

Comment:
is `always` the desired behaviour?

What if the buildkite action fails for any reason?

Reply:
Buildkite pipeline will be the one notifying if things went wrong or fine. This slack will just notify that something has been triggered in BK regardless of the status. I could split in two different notifications, but somehow I though `buildkite action` will never fail. but it could if wrong parameters.

Let me put the logic in place to play safe

Reply:
it could also fail if there was a network error or something, right?

Reply:
See 161568e